### ISSUE TITLE
Upgraded to Django 3.0 maybe? 

### ISSUE THREAD

**Issue by louisluhao** [heroku/heroku-django-template#103]

**Comment by niniack** [comment]
Agreed! @Project Maintainers If PRs for this are welcome, I'd like to take a shot at this!

**Comment by hkennyv** [comment]
+1. Would love to see this. I'd be open to contributing as well since it looks like the last commit was in 2018

**Comment by niniack** [comment]
I tried it out, seems that nothing actually had to be changed to reflect the move to Django 3. The comments/about/README just have to change to show that. There was virtually no difference in the template than some other projects I've done. 

The template seemed to make use of the package dj-database-url but this was missing in the Pipfile, which led to missing module error when deploying to Heroku. Threw that in there. Tested with the changes I made, works as expected!
